daemon/src/monitor.rs

```diff
@@ -576,7 +575,7 @@ fn map_cets(
                              tx, range, n_bits, ..
                          }| Cet {
                             txid: tx.txid(),
-                            script: script_pubkey.clone(),
+                            script: tx.output[0].script_pubkey.clone(),
                             range: range.clone(),
                             n_bits: *n_bits,
```

Comment:
Can we have a `From` for this now?

Reply:
What is `tx.output[0]`?

Reply:
> Can we have a From for this now?

Done with https://github.com/comit-network/hermes/pull/281/commits/70107f4d8131663a665b7fcc01f8bf74a44f4383.